genesyscloud/telephony_providers_edges_site/genesyscloud_telephony_providers_edges_site_init_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 
 	gcloud "terraform-provider-genesyscloud/genesyscloud"
-	telephony "terraform-provider-genesyscloud/genesyscloud/telephony"
+	telephony "terraform-provider-genesyscloud/genesyscloud/telephony_provider_edges_trunkbasesettings"
```

Comment:
Do you want to rename `telephony` to trunkbasesettings?

Reply:
yes